.github/workflows/run-tests.yaml

```
@@ -53,7 +44,7 @@ jobs:
       env:
         FORCE_COLOR: 1
         PYTEST_CI_ARGS: --cov-report=xml --cov-report=html --junitxml=test_artifacts/test_report.xml --color=yes
-      run: tox ${{ matrix.python-version == '3.6' && '--skip-missing-interpreters=true' || '' }}
+      run: tox --skip-missing-interpreters=true
```

```suggestion
      run: tox
```

That was only a py3.6/CI thing I added. Should be able to go away